models/clip/clip_src/simple_tokenizer.py

```diff
@@ -0,0 +1,132 @@
+import gzip
+import html
+import os
+from functools import lru_cache
+
+# import ftfy
```

Comment:
But how did you make it work? 😆

Reply:
In fact `ftfy` is just a text preprocessing tool that filters out some special characters, and since our users are unlikely to input strange characters, removing this should not be a big problem.